knowledge/cars/delorean/qna.yaml

```diff
@@ -0,0 +1,152 @@
+version: 2
+created_by: SEHANTA, arxspecialist, cpgill1, jessygluekler, joebelford, mattbrown1477,
+  mguginf, nsthompson, redhatconfuser1, zsplaingard
+domain: automobile
+task_description: |
+  Answer questions about the DMC DeLorean sports car
+seed_examples:
+- answer: Yes, there is only 1 of 2 prototypes still in existence. The remaining
+    existing prototype has been restored and is displayed at the AACA Museum in
+    Hershey, Pennsylvania. The 2nd prototype was sent to Lotus Cars for development
+    and evaluation and has been reported destroyed.
+  question: Are there any Delorean prototypes still in existence?
+- answer: DeLoreans, early-production models in particular, suffered from poor
+    build quality as well as mechanical issues. Early-production cars needed as
+    much as 200 hours of work at DMC Quality Assurance Centers prior to being
+    shipped to dealerships for delivery. DMC eventually sent 30 factory workers
+    to the quality centers in the US to learn about the problems and how to fix
+    them. Quality did improve over time, and by 1982 many of the quality issues
+    had been resolved. A total of four recalls were issued by the factory
+    to correct problems such as a sticking throttle, front-suspension issues and
+    an inertia switch.
+  question: Did DeLorean have quality problems?
+- answer: As much as 200 hours of work at DMC Quality Assurance Centers were required
+    prior to shipping cars to dealerships for delivery to customers.
+  question: Did DeLorean suffer from quality control issues?
+- answer: The top speed of the DeLoran DMC was tested to be 110 mph, not as fast
+    as the claimed value of 130 mph.
+  question: How fast was the DeLorean?
+- answer: The DeLorean has a four-wheel independent suspension with coil springs,
+    and telescopic shock absorbers. The front suspension uses double wishbones,
+    while the rear is a multi-link setup.
+  question: What is the suspension setup of the DMC DeLorean?
+- answer: "DMC's comparison literature noted that the DeLorean could achieve 0\u2013\
+    60 miles per hour (0\u201397 km/h) in 8.8 seconds when equipped with a manual\
+    \ transmission, but other sources indicate an acceleration time of 9.5 seconds.\
+    \ When equipped with a manual transmission, the DeLorean accelerated from\
+    \ 0 to 60 miles per hour (0 to 97 km/h) in 10.5 seconds as tested by Road\
+    \ & Track magazine. Top speed was claimed at 130 mph (209 km/h), but was tested\
+    \ to be a lacking 110 mph (177 km/h). Car & Driver put the top speed of a\
+    \ 5-speed manual at 117 mph (188 km/h) (indicated). The car was described\
+    \ as \"not quick for a sports/GT car in this price category\" by Road & Track."
+  question: What is the performance of the DMC DeLorean?
```

Comment:
The speed and acceleration figures here are from Wikipedia but are not present in the Markdown

https://github.com/instructlab/taxonomy/pull/1156#issuecomment-2146928464

Reply:
@juliadenham updated the Markdown in commit 0feadeec8794767f2ea89b6970ba78936bd1f505